providers/openfeature-sdk-meta_provider/README.md

```diff
@@ -0,0 +1,54 @@
+# OpenFeature::SDK::Provider::MetaProvider for Ruby
+
+The `OpenFeature::SDK::Provider::MetaProvider` is a utility provider implementation that takes multiple [providers](https://docs.openfeature.dev/docs/specification/sections/providers) for use during flag resolution.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'openfeature-meta-provider'
+```
+
+And then execute:
+
+```sh
+bundle install
+```
+
+Or install it yourself as:
+
+```sh
+gem install openfeature-meta-provider
+```
+
+## Usage
+
+The `MetaProvider` is initialized with a collection of `Provider`s and a strategy for fetching flags from them.
+
+```ruby
+# Create a MetaProvider
+meta_provider = OpenFeature::SDK::Provider::MetaProvider.new(
+  providers: [
+    OpenFeature::SDK::ProviderInMemoryProvider.new,
```

Comment:
Does the in-memory provider return an error if a flag isn't found? I just want to confirm that this example should actually work 😄

Reply:
Yep! We'll return back the `FLAG_NOT_FOUND` error code. [Ref](https://github.com/open-feature/ruby-sdk/blob/4ec0326ddabed49485200abbd6f1597b8654b573/lib/open_feature/sdk/provider/in_memory_provider.rb#L52)